README.md

```diff
@@ -289,6 +292,14 @@ expect(getByTestId('count-value')).not.toHaveTextContent('21')
 // ...
 ```
 
+## Other
+
+#### `waitForExpect(expectation: () => void, timeout?: number, interval?: number) => Promise<{}>;`
```

Comment:
Let's actually put this right after `flushPromises` and call out a limitation of `flushPromises`. Specifically that it only flushes the promises that have been queued up already and not those that will be queued up subsequently (maybe we can provide an example?)